src/lib.rs

```diff
@@ -22,13 +22,13 @@ mod node;
 
 const DEPTH: u32 = 7;
 const WHITE: u8 = 2;
-const BLACK: u8 = 1;
 
 #[pymodule]
 fn rust_ext(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
     #[pyfn(m, "get_next_move")]
     /// Interfacing function.
     /// Takes the Python GIL, the board in the shape of a 19*19 numpy 2d array, the color of the human player, a boolean that indicates if this is a hint request, and the number of captures made by the human and the ai.
+    #[allow(unused_variables)]
```

Comment:
why?

Reply:
Because the parameter `hint` is never used, and I only wanted to get rid of the associated warning without having to touch de python part. I'm open to suggestions about that BTW.

Reply:
It should be used, and it should swap player and enemy, it's not the case ?